docker.go

```diff
@@ -514,19 +516,76 @@ type DockerProvider struct {
 
 var _ ContainerProvider = (*DockerProvider)(nil)
 
+// or through Decode
+type TestContainersConfig struct {
+	Host      string `properties:"docker.host"`
```

Comment:
Here we are setting host as required. Is it correct?

Reply:
ah, I missed this one. The file may exist but `docker.host` can be missing in it, so I would recommend making it optional as well 👍